lib/rubocop/cop/layout/line_continuation_leading_space.rb

```diff
@@ -51,6 +51,10 @@ class LineContinuationLeadingSpace < Base
         private_constant :LINE_1_ENDING, :LINE_2_BEGINNING,
                          :LEADING_STYLE_OFFENSE, :TRAILING_STYLE_OFFENSE
 
+        def self.autocorrect_incompatible_with
```

Comment:
You might want to add this as a comment here:

> When both cops are activated and run in the same iteration of the correction loop, Style/StringLiterals undoes the moving of spaces that Layout/LineContinuationLeadingSpace performs. This is because StringLiterals takes the original string content and transforms it, rather than just modifying the delimiters, in order to handle escaping for quotes within the string.

Reply:
@bbatsov added the comment, thanks!